src/pages/Models/Models.js

```diff
@@ -131,20 +136,28 @@ export default function Models() {
         </div>
       </Header>
 
-      <div className="l-content">
-        <div className="models">
-          <StatusStrip
-            statusList={{
-              model: [
-                { label: "Blocked", count: blocked },
-                { label: "Alert", count: alert },
-                { label: "Running", count: running },
-              ],
-            }}
-          />
-          <ModelTableList groupedBy={groupModelsBy} filters={filters} />
+      {modelsLoaded ? (
+        <FadeIn isActive={modelsLoaded}>
+          <div className="l-content">
+            <div className="models">
+              <StatusStrip
+                statusList={{
+                  model: [
+                    { label: "Blocked", count: blocked },
+                    { label: "Alert", count: alert },
+                    { label: "Running", count: running },
+                  ],
+                }}
+              />
+              <ModelTableList groupedBy={groupModelsBy} filters={filters} />
+            </div>
+          </div>
+        </FadeIn>
+      ) : (
+        <div className="model-details__loading">
```

Comment:
The spinner could also fade in too?

Reply:
Tried this.. visual fade in for loading is very difficult to see, not worth added complexity in code.